CONTRIBUTING.md

```diff
@@ -245,3 +245,34 @@ npm run update_prebuilt
 
 That last step is easy to forget, so try to make it a habit whenever you're
 prepping a PR that includes frontend work.
+
+### MacOS
+
+Install Xcode and Homebrew Mac. Once that is done: 
```

Comment:
Might be good to provide a few words of explanation and make Xcode and Homebrew hyperlinks to their respective landing pages. Is Xcode just a dep for Homebrew, or do you need it in order to get git as well? If it's just for Homebrew, then we might not need to list it here.